lib/markers.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Markers do
```

Comment:
I don't think the `get_` prefix adds any value.

`foo = Markers.all()` and 
`marker = Marker.from_symbol(player_symbol)` convery enough information and are closer to regular sentences.